pkg/translate/transform.go

```diff
@@ -0,0 +1,91 @@
+package translate
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.expect.digital/translate/pkg/model"
+	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
+)
+
+// ----------------------Common types----------------------
+
+// uuidToProto converts uuid.UUID to string.
+func uuidToProto(u uuid.UUID) string {
+	if u == uuid.Nil {
+		return ""
+	}
+
+	return u.String()
+}
+
+// uuidFromProto converts string to uuid.UUID.
+func uuidFromProto(s string) (uuid.UUID, error) {
+	if s == "" {
+		return uuid.Nil, nil
+	}
+
+	id, err := uuid.Parse(s)
+	if err != nil {
+		return uuid.Nil, fmt.Errorf("parse uuid: %w", err)
+	}
+
+	return id, nil
+}
+
+// ----------------------Service----------------------
+
+// serviceToProto converts model.Service to translatev1.Service.
+func serviceToProto(s *model.Service) *translatev1.Service {
+	return &translatev1.Service{Id: uuidToProto(s.ID), Name: s.Name}
```

Comment:
Cover the case when s is nil when transforming models.

```suggestion
	if s == nil {
		return nil
	}
	
	return &translatev1.Service{Id: uuidToProto(s.ID), Name: s.Name}
```